data/cadastre/plss/index.html

```diff
@@ -363,13 +364,10 @@ <h3 id="QuarterSection">PLSS Quarter Sections GCDB</h3>
       derived from the Intersected layer of the CadNSDI published version of the PLSS, which includes quarter-quarter,
       sixteenth or government lot division, mineral surveys, and many Unsurveyed areas of the PLSS. Many areas will not
       have a designated location (NE, NW, SW, SE) in the original data so will not be classified in the Quarter Sections
-      layer.This data is from the Utah PLSS Fabric Version 2.1 2019 of the Utah GCDB . Updated Jan. 15, 2019</p>
-    <p>Updates were made to Quarter Sections in Utah County to add quarter sections to areas that were not broken down
-      less than the Section level. They were added, using information from points collected by county surveyors, that is
-      in the data and extrapolating information from adjoining sections wherever possible. They are still areas that
-      could not be interpreted well enough and they were left empty, beyond the section level.
-      In the future this data will continue to be carried in this feature class but not incorporated into the official
-      PLSS Fabric.</p>
+      layer.This data is from the Utah PLSS Fabric Version 2.3 2020 of the Utah GCDB.</p>
+    <pQuarter Sections were consolidated from the Quarter Quarter Sections to better represent the four quarters in all Aliquot Part areas. 
+      Areas with Lots or Special Surveys remain as is. 
+      Updated 4/8/2022</p>
```

Comment:
```suggestion
      layer. This data is from the Utah PLSS Fabric Version 2.3 2020 of the Utah GCDB.</p>
    <p>Quarter Sections were consolidated from the Quarter Quarter Sections to better represent the four quarters in all Aliquot Part areas. Areas with Lots or Special Surveys remain as is. Updated 4/8/2022</p>
```
Fixed the formatting

Reply:
That sounds fine, go ahead and change it.